### PR TITLE
created the delete venue

### PR DESCRIPTION
**Pull request by kaamil2**
Screenshot 2024-09-28 at 4 39 50 PM
Screenshot 2024-09-28 at 4 39 44 PM
Screenshot 2024-09-28 at 4 39 36 PM
Screenshot 2024-09-28 at 4 39 33 PM
# Description

[Link to Ticket](insert the link to your ticket inside the parenthesis here)

Please include a summary of the changes and the related issue. Please also
include relevant motivation, context, and images!

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. If they are unit
tests, provide the file name the tests are in. If they are not unit tests,
describe how you tested the change.

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by CamPlume1 — APPROVED** [review]
Looks Great

**Review by CamPlume1 — APPROVED** [review]

File: backend/internal/handlers/venues/venues.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+func (s *Service) DeleteVenue(c *fiber.Ctx) error {
+	fmt.Println("Pinging correct service")
+
+	venueID := c.Params("venueId")
```
Comment (garrettladley) [inline backend/internal/handlers/venues/venues.go:14]: you can simplify this block by deferring the length check to [`uuid.Parse`](https://github.com/google/uuid/blob/v1.6.0/uuid.go#L68)

File: backend/internal/handlers/venues/venues.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	if venueID == "" {
+		c.Status(http.StatusBadRequest)
```
Comment (garrettladley) [inline backend/internal/handlers/venues/venues.go:16]: for consistency, use the err helper functions from [`errs`](https://github.com/GenerateNU/nightlife/blob/main/backend/internal/errs/api_err.go). if there isn't one that suits your needs, create a new one!

File: backend/internal/handlers/venues/venues.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		return fiber.NewError(400)
+	}
+	venueIDFormatted, err := uuid.Parse(venueID)
+
+	if err != nil {
+		c.Status(http.StatusBadRequest)
+		return fiber.NewError(400, "Delete failed due to malformed venue ID")
+	}
+
+	venueDelete := s.store.DeleteVenue(c.Context(), venueIDFormatted)
```
Comment (garrettladley) [inline backend/internal/handlers/venues/venues.go:26]:
```suggestion
	err = s.store.DeleteVenue(c.Context(), venueIDFormatted)
```

File: backend/internal/handlers/venues/venues.go
Comment (garrettladley) [inline backend/internal/handlers/venues/venues.go:1]: this func will be a little "noisy" with the `fmt.Println` calls, could you remove?

File: backend/internal/handlers/venues/venues.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+	if err != nil {
```
Comment (garrettladley) [inline backend/internal/handlers/venues/venues.go:28]: what err is this checking?

File: backend/internal/storage/postgres/profiles.go [hunk 5]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/google/uuid"
 )
 
-func (db *DB) UpdateProfilePrefences(ctx context.Context, userID uuid.UUID, preferencetypeto string, preferencevalueto string, preferenceType string, preferenceValue string) error {
+func (db *DB) UpdateProfilePrefences(ctx context.Context, userid uuid.UUID, preferencetypeto string, preferencevalueto string, preferenceType string, preferenceValue string) error {
```
Comment (garrettladley) [inline backend/internal/storage/postgres/profiles.go:11]: why change the casing?

**Review by garrettladley — COMMENTED** [review]
some nits then LGTM!

**Pull request closed without merging**